pylatex/table.py

```diff
@@ -237,6 +300,153 @@ class LongTable(Tabular):
 
     packages = [Package('longtable')]
 
+    header = False
+
+    def end_table_header(self):
+        r"""End the table header which will appear on every page."""
+
+        if self.header:
+            msg = "Table already has a header"
+            raise TableError(msg)
+
+        self.header = True
+
+        self.append(NoEscape(r'\endhead'))
+
 
 class LongTabu(LongTable, Tabu):
     """A class that represents a longtabu (more flexible multipage table)."""
+
+
+class ColoredTabu(Tabu):
```

Comment:
There is no need to create new classes for this. 
Just add the functionality to Tabular.